document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapper/smallfull/model/ixbrl/balancesheet/otherliabilitiesandassets/OtherLiabilitiesAndAssets.java

```diff
@@ -0,0 +1,158 @@
+package uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.Gson;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.AccrualsAndDeferredIncome;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.CreditorsAfterMoreThanOneYear;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.CreditorsDueWithinOneYear;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.NetCurrentAssetsLiabilities;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.PrepaymentsAndAccruedIncome;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.ProvisionsForLiabilities;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.TotalAssetsLessCurrentLiabilites;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.TotalNetAssets;
+
+import java.util.Objects;
+
+public class OtherLiabilitiesAndAssets {
+
+    @JsonProperty("prepayment_and_accrued_income")
```

Comment:
This field does not match small-full-accounts template. This should be `prepayments_and_accrued_income`

Reply:
done